scala-package/core/src/main/scala/ml/dmlc/mxnet/module/DataParallelExecutorGroup.scala

```diff
@@ -312,8 +312,12 @@ class DataParallelExecutorGroup private[module](
     if (labelShapes != None) decideSlices(labelShapes.get)
     else null
 
-  private val outputLayouts = symbol.listOutputs().map(name =>
-    DataDesc.getBatchAxis(symbol.get(name).attr("__layout__"))
+  private val outputLayouts = symbol.listOutputs().map(name => {
+    val sym = symbol.get(name)
```

Comment:
`symbol.get` returns a [new Symbol value](https://github.com/apache/incubator-mxnet/blob/master/scala-package/core/src/main/scala/ml/dmlc/mxnet/Symbol.scala#L99-L103) each time. Since that Symbol falls out of scope here, we can't reuse it. I believe because a new handle is retrieved in `symbol.get` that this shouldn't cause an issue elsewhere.